app/src/main/java/com/lftechnology/unito/activity/MainActivity.java

```diff
@@ -0,0 +1,312 @@
+import android.os.Bundle;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.support.v7.widget.helper.ItemTouchHelper;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.animation.Animation;
+import android.view.animation.RotateAnimation;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import com.google.gson.Gson;
+import com.lftechnology.unito.utils.OnKeyEvents;
+import com.lftechnology.unito.R;
+import com.lftechnology.unito.adapter.DrawerRecyclerViewAdapter;
+import com.lftechnology.unito.bus.EventBus;
+import com.lftechnology.unito.bus.FlingListener;
+import com.lftechnology.unito.bus.NavigationMenuChangeDetails;
+import com.lftechnology.unito.bus.ScrollListener;
+import com.lftechnology.unito.bus.SwapFragment;
+import com.lftechnology.unito.constant.AppConstant;
+import com.lftechnology.unito.fragment.MainFragment;
+import com.lftechnology.unito.helper.OnStartDragListener;
+import com.lftechnology.unito.helper.SimpleItemTouchHelperCallback;
+import com.lftechnology.unito.utils.GeneralUtils;
+import com.lftechnology.unito.utils.SoftKeyBoard;
+import com.squareup.otto.Subscribe;
+
+import java.util.Arrays;
+
+public class MainActivity extends AppCompatActivity implements AdapterView.OnItemSelectedListener, OnStartDragListener, OnKeyEvents {
+    private Toolbar toolbar;
+    private LinearLayout mToolbarContainer;
+    private String mSelectedConversion;
+    private DrawerLayout mDrawerLayout;
+    private ActionBarDrawerToggle mDrawerToggle;
+
+    private RecyclerView mRecyclerView;
+    private DrawerRecyclerViewAdapter mAdapter;
+    private RecyclerView.LayoutManager mLayoutManager;
+    private String[] mDrawerRecyclerViewDataset;
+    private ItemTouchHelper mItemTouchHelper;
+    private LinearLayout linearLayout;
+    private int DY = 5;
+    private boolean spinDirection = true;
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        EventBus.register(this);
+    }
+
+    @Override
+    public void onPause() {
+        super.onPause();
+        EventBus.unregister(this);
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        mToolbarContainer = (LinearLayout) findViewById(R.id.toolbarContainer);
```

Comment:
@grishmashrestha why are you initializing views the old fashioned way? I saw butter knife on your dependencies. Use that.